### PR TITLE
Fix 102 tod os voznyarskyy

### PR DESCRIPTION
**Pull request by VozNick**
working database migration. Remove constraint key (person) from restaurant table. Add person id field in restaurant entity.

### CODE REVIEW

File: server/pom.xml [hunk 1]
```diff
@@ -29,56 +29,63 @@
...
-     <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-jpa</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-security</artifactId>
+    </dependency>
+
```
Comment (vitaliimak) [inline server/pom.xml]: pom.xml file can have more than 1000 lines, do you want to reformat it all?

**Review by vitaliimak — CHANGES_REQUESTED** [review]
I've created DB in dev branch, then switch to this branch and got errors
[24.04.2020-spring.log](https://github.com/Social-projects-Rivne/Lunch_Time/files/4526982/24.04.2020-spring.log)

**Comment by vitaliimak** [comment]
Your task doesn't place in `On Verification`

**Comment by VozNick** [comment]
@Vitalii Makarchenkov last week I was doing migration of an existing database from dev. I've wrote a script (V1) for that db (baseline). After that I edited one table (restaurant) and created another script (V2). 
Meanwhile in dev were created another two tables. And I need to make merge dev first to open PR.

I created another script (V3) and it works on my local.
So how to resolve this... ? Do I need to create again this base line from dev, and again to make changes to restaurant table and make changes in project?

**Comment by vitaliimak** [comment]
@VozNick unfortunately, I can't help you. I don't have enough experience in migration

**Comment by narekacy** [comment]
@VozNick Despite of my experience in migration since 1993, I'm not sure if this is exactly what you are looking for, but,

you should prepare your PR in condition that it will be suitable with current dev right now. 

And be careful with mentioning with '@'. That one is another Vitalik

**Comment by VozNick** [comment]
Everything is working. Please check. Thanks

**Comment by VozNick** [comment]
pom.xml is from dev, but in git it shows that it has many changes (because of previous commits)).
I just added flyway dependancy

**Comment by vitaliimak** [comment]
image
But can in staging

**Comment by VozNick** [comment]
can't add feedback on staging too..

**Comment by VozNick** [comment]
is it really an essue in my task?

**Comment by vitaliimak** [comment]
@VozNick it's happened because of wrong login. Your code works. And you should logout and login in staging to send feedback

**Review by vitaliimak — APPROVED** [review]

**Review by r-yevchuk — APPROVED** [review]

**Review by Melnyk9506 — APPROVED** [review]

**Pull request merged**